libcalico-go/lib/clientv3/globalnetworkpolicy_e2e_test.go

```diff
@@ -155,7 +156,7 @@ var _ = testutils.E2eDatastoreDescribe("GlobalNetworkPolicy tests", testutils.Da
 			Expect(outError.Error()).To(Equal("resource already exists: GlobalNetworkPolicy(" + tieredGNPName(name1, tier) + ")"))
 
 			By("Getting GlobalNetworkPolicy (name1) and comparing the output against spec1")
-			res, outError := c.GlobalNetworkPolicies().Get(ctx, name1, options.GetOptions{})
+			res, outError := c.GlobalNetworkPolicies().Get(ctx, tieredGNPName(name1, tier), options.GetOptions{})
```

Comment:
I'm a bit surprised to see all of these test changes - I would have expected this change to be compatible with our existing v3 client tests?